.github/workflows/draft-release.yaml

```diff
@@ -0,0 +1,77 @@
+name: "Draft new release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      new-version:
+        type: choice
+        description: "Which version you'd like to release?"
+        options:
+        - major
```

Comment:
This is fine for `-rc.2` but the workflow a;sp needs to allow to go from version -> version+1-rc.1 -...> version+1-rc.N -> version+1

Reply:
I'll look into it in the next PR! thanks for pointing out!